Automattic/newspack-blocks
# Add autocomplete search to article block story select

### All Submissions:






### Changes proposed in this Pull Request:



I wanted to get this in before the work to bring the articles block to wpcom gets too far along, because this introduces a backwards-incompatible change.

This PR changes up the way the specific stories selection works. Instead of picking one post from a select dropdown of the last 50 posts, the user can pick up to 100 posts using autocomplete search:

<img width="1108" alt="Screen Shot 2019-11-15 at 6 56 28 AM" src="https://user-images.githubusercontent.com/7317227/68952678-1bf79580-0775-11ea-9c94-76674689cec5.png">

### How to test the changes in this Pull Request:

1. Add an articles block, toggle "Choose specific stories" on, and select some stories. The order of the stories in the block should match the order of the stories selected. Save and view on frontend. Verify things look the same, and the stories you requested are available in the correct order.
2. Verify tag/author/category search stuff still works on homepage blocks, and carousel block still works.

### Other information:
